src/tests/unit/UserInfoHandler.test.ts

```diff
@@ -0,0 +1,35 @@
+import { lambdaHandler } from "../../UserInfoHandler";
+import { mock } from "jest-mock-extended";
+import { VALID_USERINFO } from "./data/userInfo-events";
+import { UserInfoRequestProcessor } from "../../services/UserInfoRequestProcessor";
+import { HttpCodesEnum } from "../../models/enums/HttpCodesEnum";
+import { CONTEXT } from "./data/context";
+
+const mockedUserInfoRequestProcessor = mock<UserInfoRequestProcessor>();
+
+jest.mock("../../services/UserInfoRequestProcessor", () => {
+	return {
+		UserInfoRequestProcessor: jest.fn(() => mockedUserInfoRequestProcessor),
+	};
+});
```

Comment:
Don't think we need this mock when we do `mockReturnValue` in every test

Reply:
Removed